src/coreclr/src/inc/sospriv.idl

```diff
@@ -390,3 +390,14 @@ interface ISOSDacInterface7 : IUnknown
     HRESULT GetProfilerModifiedILInformation(CLRDATA_ADDRESS methodDesc, struct DacpProfilerILData *pILData);
     HRESULT GetMethodsWithProfilerModifiedIL(CLRDATA_ADDRESS mod, CLRDATA_ADDRESS *methodDescs, int cMethodDescs, int *pcMethodDescs);
 };
+
+[
+    object,
+    local,
+    uuid(3f5f88ae-a312-42c1-affa-ecedfaf248c0)
+]
+interface ISOSDacInterface8 : IUnknown
+{
+    HRESULT GetAssemblyLoaderAllocatorData(CLRDATA_ADDRESS loaderAllocator, struct DacpAssemblyLoaderAllocatorData* pAssemblyLoaderAllocatorData);
+    HRESULT GetCLRPrivBinderAssemblyLoadContextData(CLRDATA_ADDRESS binder, struct DacpCLRPrivBinderAssemblyLoadContextData* pBinderData);
```

Comment:
`CLRPrivBinder` is a poor internal name. I would use the public facing name for this, ie AssemblyLoadContext.

Reply:
Sounds reasonable.